modules/darwinConfigurations.macos-01/default.nix

```diff
@@ -1,14 +1,14 @@
 { self, lib, inputs, config, ... }: {
   perSystem = { config, self', inputs', pkgs, ... }: {
-    apps."deploy-${(import ./attrs.nix).hostName}" = {
+    apps."deploy-macos-01" = {
```

Comment:
can we use a variable for DRY?

Reply:
fixed via https://github.com/holochain/holochain-infra/pull/5/commits/484666de5207337180e9bfa70280883d4fa0e798
Those apps are now automatically generated for all hosts in the repo